spec/support/vcr_cassettes/caseflow/appeal_with_null_issue_description.yml

```diff
@@ -0,0 +1,100 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://dsva-appeals-certification-dev-1895622301.us-gov-west-1.elb.amazonaws.com/api/v2/appeals
```

Comment:
Please add the URI to the [vcr config](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb).